### PR TITLE
290 faq entry to point to nav customization config options

### PR DESCRIPTION
**Pull request by llewellyn-sl**

### CODE REVIEW

File: docs/pipeline-schema/overview.md [hunk 1]
```diff
@@ -6,75 +6,41 @@ description: 'A brief introduction to Pipeline Schema.'
...
 
-You can create a UI for your pipeline parameters using the **pipeline schema**.
-
-
-## Why do you need a schema file for your pipelines or software applications?
-
-Pipeline schema file is used to describe the different paraments used by the Nextflow workflow and the input parameters that the pipeline accepts.
-
-Nextflow Tower uses this file to automatically generate the pipeline inputs form and validate the user provided parameters in a user friendly way.
+Tower uses pipeline schema to build the launchpad parameters form.
```
Comment (gwright99) [inline docs/pipeline-schema/overview.md:18]: This looks like it's slightly off grammatically. Are we missing a qualifying word here (e.g. "a", "your repo's")? 

i.e. "Tower uses your repo's pipeline schema to build a bespoke launchpad parameters form."

File: docs/pipeline-schema/overview.md [hunk 2]
```diff
@@ -6,75 +6,41 @@ description: 'A brief introduction to Pipeline Schema.'
...
 
 ## How can I build my own Pipeline schema file for my Nextflow pipelines?
 
-The pipeline schema is based on [json-schema.org](https://json-schema.org/) syntax, 
-therefore it can be written with a simple text editior, even though can difficult for complex pipelines.
+The pipeline schema is based on [json-schema.org](https://json-schema.org/) syntax, with some additional conventions. This can get complex, so we do not recommend creating or editing this file manually.
 
-The [nf-core](https://nf-co.re/) project provides an handy tool that helps writing the schema 
-file by running the `nf-core schema build` command in the pipeline root directoty.
+Instead, we recommmend using tools from the [nf-core](https://nf-co.re/) project, which provides a toolset for developing Nextflow pipelines.
```
Comment (gwright99) [inline docs/pipeline-schema/overview.md:28]: Better to link the [schema command](https://nf-co.re/tools/#pipeline-schema) directly? 

I know the sentence recommends NFCORE tooling in general, but this section is specifically about nextflow schema.

Comment (gwright99) [reply docs/pipeline-schema/overview.md:28]: Oh, I see, it's just below ...

**Review by gwright99 — APPROVED** [review]

File: docs/pipeline-schema/overview.md (hunk 1 above)
Comment (llewellyn-sl) [inline docs/pipeline-schema/overview.md:18]:
```suggestion
Tower uses your pipeline schema to build a bespoke launchpad parameters form.
```

**Pull request merged**